sympy/parsing/latex/_antlr/latexlexer.py

```diff
@@ -331,15 +349,16 @@ class LaTeXLexer(Lexer):
                   u"FUNC_ARCSEC", u"FUNC_ARCCOT", u"FUNC_SINH", u"FUNC_COSH",
                   u"FUNC_TANH", u"FUNC_ARSINH", u"FUNC_ARCOSH", u"FUNC_ARTANH",
                   u"FUNC_SQRT", u"CMD_TIMES", u"CMD_CDOT", u"CMD_DIV", u"CMD_FRAC",
-                  u"CMD_MATHIT", u"UNDERSCORE", u"CARET", u"COLON", u"WS_CHAR",
-                  u"DIFFERENTIAL", u"LETTER", u"DIGIT", u"NUMBER", u"EQUAL",
-                  u"LT", u"LTE", u"GT", u"GTE", u"BANG", u"SYMBOL" ]
+                  u"CMD_BINOM", u"CMD_DBINOM", u"CMD_TBINOM", u"CMD_MATHIT",
+                  u"UNDERSCORE", u"CARET", u"COLON", u"WS_CHAR", u"DIFFERENTIAL",
+                  u"LETTER", u"DIGIT", u"NUMBER", u"EQUAL", u"LT", u"LTE",
+                  u"GT", u"GTE", u"BANG", u"SYMBOL" ]
 
     grammarFileName = u"LaTeX.g4"
 
     def __init__(self, input=None, output=sys.stdout):
         super(LaTeXLexer, self).__init__(input, output=output)
-        self.checkVersion("4.7.1")
+        self.checkVersion("4.7.2")
```

Comment:
I see this always gets modified across different versions.
I have rebuilt with 4.7.1 and I see some diffs.

Reply:
The latest version `4.8` came out, do you want me to upgrade to that, seems to work fine locally.
or should I stick with `4.7.2` in which case I'll update `$ conda install -c conda-forge antlr=4.7` in `README.md`

Reply:
I downgraded to 4.7.1 and yeah there were some differences. 😄